src/sandstorm/web-session.capnp

```diff
@@ -190,6 +198,28 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
     # semantically equivalent
   }
 
+  struct Range {
+    # Specifies a range of bytes within a larger blob. Used for `Range` and `Content-Range`
+    # headers.
+
+    start @0 :UInt64;
+    # Start point of the range, in bytes.
+
+    end :union {
+      # Endpoint of the range, in bytes.
+
+      endOfContent @1 :Void;
+      position @2 :UInt64;
+    }
+
+    fullSize :union {
```

Comment:
As far as I can tell, this is only valid in responses. Maybe we should remove it, since we only currently use the Range struct in requests?

Reply:
Huh. It's been a long time since I wrote this, but I wonder if I had intended to include a `Range` under `Response.content` but forgot... otherwise I'm not sure how the `Content-Range` header is supposed to be constructed for the response.

Reply:
See the comment you wrote below -- looks like the idea was to just insist that the app returns the range that was actually asked for, so the supervisor only needs to check the status code.

Reply:
But how is the overall size communicated? I guess the final `Content-Range` header would always end up ending in `/*`?

BTW, what makes you say that `fullSize` is not valid for requests? It doesn't appear in a GET request (`Range` header), but a PUT that is uploading a range (`Content-Range` header) could include it, right? (However, I'm not sure if it's actually worth supporting range PUTs...)

Reply:
Yes, I was only thinking about GET requests. I guess I don't feel that strongly about this.